components/brave_wallet_ui/stories/screens/onboarding.tsx

```diff
@@ -160,11 +190,20 @@ function Onboarding (props: Props) {
         {isImporting &&
           <OnboardingImportMetaMaskOrLegacy
             onSubmit={onImport}
+            onImportPasswordChanged={handleImportPasswordChanged}
             onPasswordChanged={handlePasswordChanged}
-            disabled={password === ''}
+            onConfirmPasswordChanged={handleConfirmPasswordChanged}
+            disabled={checkConfirmedPassword || checkPassword || password === '' || confirmedPassword === '' || importPassword === ''}
```

Comment:
Let's have a separate variable for this.

Reply:
Done!